src/containers/Messages$/Messages$.js

```diff
@@ -11,21 +11,25 @@ import ContentPanel from '../../components/ContentPanel'
 
 class Messages$ extends React.Component {
 
-  constructor() {
+  constructor(props) {
```

Comment:
I want to ask what was the purpose of keeping the geo data as this component's state. I assume you had genuine reason for this.